src/actions/userActions.js

```diff
@@ -0,0 +1,22 @@
+import axios from 'axios';
+import { SET_SIGNUP_FORM_INPUT, SET_SUBMIT_USERS, USER_ERRORS } from './type';
+
+export const setSignupInput = data => async (dispatch) => {
+  const url = '/api/users';
+  return axios.post(url, data).then((res) => {
+    dispatch({
+      type: SET_SIGNUP_FORM_INPUT,
+      payload: res.data
+    });
+  }).catch((error) => {
+    const { data } = error.response;
+    dispatch({
+      type: USER_ERRORS,
+      payload: data
+    });
+  });
+};
+export const setSubmitUsers = payload => ({
```

Comment:
What is the use of this function? Are you calling it anywhere?

Reply:
@cop1fab because this file is in actions folder, it is not necessary to name it `userActions.js`, I think  `user.js` is enough